### ISSUE TITLE
Mismatch in cluster URLs

### ISSUE THREAD

**Issue by xcoulon** [fabric8-services/fabric8-cluster#66]
when a cluster is returned by one of the `show`/`list` endpoints, the URLs have a trailing slash added to them, even if it did not ended with a trailing slash in the database.
But if the URL returned by the endpoint is later used to link/unlink an identity, then the look-up will fail with a `400` error (although it could be a `404` instead).

Possible solutions:
- stop adding trailing slashes when returning cluster info, i.e., return the data as-is
- include the trailing slash when creating/updating the cluster info in the database, and return the data as-is (but we should migrate all existing data to ensure that all URLs have such a trailing slash)

**Comment by alexeykazakov** [comment]
We should make sure we either always add a trailing slash to the URLs OR we always do not add them when saving them in DB. And we should add or remove the trailing slash in the request URL when doing an lookup accordingly.

**Comment by xcoulon** [comment]
> We should make sure we either always add a trailing slash to the URLs OR we always do not add them when saving them in DB. And we should add or remove the trailing slash in the request URL when doing an lookup accordingly.

yes, we could also do that. But what was the reason for adding those trailing slashes at first? Can't we just trust the user's input?
